src/rust/src/wallet.rs

```diff
@@ -150,7 +149,8 @@ pub struct Wallet {
     nullifiers: BTreeMap<Nullifier, OutPoint>,
     /// The incremental Merkle tree used to track note commitments and witnesses for notes
     /// belonging to the wallet.
-    commitment_tree: BridgeTree<MerkleHashOrchard, u32, NOTE_COMMITMENT_TREE_DEPTH>,
+    // TODO: Replace this with an `orchard` crate constant (they happen to be the same).
+    commitment_tree: BridgeTree<MerkleHashOrchard, u32, { sapling::NOTE_COMMITMENT_TREE_DEPTH }>,
```

Comment:
```suggestion
    commitment_tree: BridgeTree<MerkleHashOrchard, u32, 32>,
```
Temporarily, it is better to hard-code than to use another constant that happens to be the same (also change the comment and specify the type of the constant if necessary).